javascript/app.js

```diff
@@ -0,0 +1,481 @@
+	const familyArray = [];
+for (let i = 0; i < arrayString.length; i++) {
+	familyArray.push(arrayString[i].length);
+	}
+	console.log(familyArray);
+	return familyArray;
+} 
+getMultipleLengths(["Ashley", "Katya", "James", "Boris", "Petrus", "Tetris the Roomba"]);
+
+//I tend to forget the return!!
+
+//G Maxofthree
+
+const maxOfThree = (number1, number2, number3) => {
+
+	if (number1 >= number2 && number1 >= number3) {
+		console.log(number1);
+		return number1;
+	} else if (number2 >= number1 && number2 >= number3) {
+		console.log(number2);
+		return number2;
+	} else  {
+		console.log(number3);
+		return number3;
+	}
+
+
+};
+maxOfThree(4, 5, 6);
+// JSHINT IS MY FRIEND
+
+
+//H printLongestWord
+/* I don't understand why this isn't working... I found a different solution but... yeah. 
+const printLongestWord = (arrayOfStrings) => {
+	let longestlength = 0;
+	for (let i = 0; i < arrayOfStrings.length; i++) {
+		if (arrayOfStrings[i] > longestlength) {
+			longestlength = arrayOfStrings[i].length;
+			break;
+		}
+	} console.log(longestlength);
+};
+*/
+/* Couldn't get this to work either... 
+
+const printLongestWord = (arrayOfStrings) => {
+	arrayOfStrings.sort(function(a, b) { return b.length - a.length; })[0];
+} return arrayofStrings;
+*/
+
+//omg I spent like ten minutes trying to diagnose this and it turned out I had lower cased the "of" in array of strings. Kept showing up as arrayOfStrings being undefined. 
+const printLongestWord = (arrayOfStrings) => {
+	let longestString = " ";
+	arrayOfStrings.forEach(function(strings) {
+		if (strings.length >= longestString.length) {longestString = strings;
+			}
+		});
+	return longestString;
+};
+console.log(printLongestWord(["Longest", "Word", "if", "You", "Please"]));
+
+//I Transmogrify
+
+const transmogrify = (number1, number2, number3) => {
+	let transmogrified = (number1 * number2) ** number3;
+	//also math is bad and it should feel bad. :( UGH, that **, you got to be kitten me. 
+	return transmogrified;
+}
+
+console.log(transmogrify(4, 5, 6)); 
+
+//J reverseWordOrder
+//without split, reverse or join??? NOOOOOOO. DU. DU HAST. DU HAST MICH. This problem is exactly like a rammstein concert with a burning cross falling on me and everything. 
+
+const reverseWordOrder = (string) => {
+	let newString = " ";
+for (let i = string.length - 1; i >= 0; i--) {
+		newString += string[i];
+
+	}
+	console.log(newString);
+	return newString;
+}
+
+reverseWordOrder("Hello");
+
+//K Get down and dirty with Math.random(). 
+
+//1
+const returnInteger = () => {
+	return Math.floor(Math.random() * 10);
+}
+//2
+const returnInteger2 = () => {
+	return Math.floor(Math.random()* 90) + 10;
+}
+//3 
+const returnInteger3 = () => {
+	return Math.floor(Math.random() * 12735) + 532;
+}
+
+//4 
+const returnInteger4 = () => {
+	return Math.floor(Math.random() * 10) + 1;
+}
+
+//5 
+const getRandomElement = (array) => {
+	return array[Math.floor(Math.random() * array.length)+ 1];
+}
```

Comment:
you're missing part of the answer.
re-read the question:
**Add a few more quotes to the quotes array from question III-B-1 above.** Write a function that will take an array as a parameter, and return a random element from that array. **Call your function a few times, passing in the quotes array.** Give it a nice semantic name like getRandomElement.